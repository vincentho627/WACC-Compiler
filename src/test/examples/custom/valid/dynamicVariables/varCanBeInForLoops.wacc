# Test case to show that var can be used in for loops

begin
    var x = 100;
    for i in 5 until 10 do
    x = x + i;
    println x
    done
end
