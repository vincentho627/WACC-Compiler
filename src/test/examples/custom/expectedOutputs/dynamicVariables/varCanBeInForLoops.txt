105
111
118
126
135
